specs/core/0_beacon-chain.md

```diff
@@ -1671,7 +1671,8 @@ Verify that `len(block.body.attestations) <= MAX_ATTESTATIONS`.
 
 For each `attestation` in `block.body.attestations`:
 
-* Verify that `attestation.data.slot <= state.slot - MIN_ATTESTATION_INCLUSION_DELAY < attestation.data.slot + SLOTS_PER_EPOCH`.
+* Verify that `attestation.data.slot <= state.slot - MIN_ATTESTATION_INCLUSION_DELAY`.
```

Comment:
```suggestion
* Verify that `attestation.data.slot + MIN_ATTESTATION_INCLUSION_DELAY <= state.slot`.
```
Reasons:
1. So we can see both the lower-bound and upper-bound of `state.slot` clearly when reading these two lines.
2. The overflow/underflow discussion. Do our best to prevent it anyway.

Reply:
done